From dd12d8c3b4aa5425be57c22bef2b29084b23d994 Mon Sep 17 00:00:00 2001
From: Martin Perina <mperina@redhat.com>
Date: Mon, 14 Apr 2014 08:36:44 +0200
Subject: [RHEL7 PATCH 5/7] Rename check_fence_kdump to check_pcs_fence_kdump

Renames check_fence_kdump to check_pcs_fence_kdump to clearly identify
that this method checks only Pacemaker configuration of fence_kdump.

Related: rhbz#1086988
Signed-off-by: Martin Perina <mperina@redhat.com>
---
 kdumpctl | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/kdumpctl b/kdumpctl
index db673e4..f7012c4 100755
--- a/kdumpctl
+++ b/kdumpctl
@@ -185,9 +185,9 @@ function check_config()
 	return 0
 }
 
-# check_fence_kdump <image timestamp>
-# return 0 if fence_kdump is configured and kdump initrd needs to be rebuilt
-function check_fence_kdump()
+# check_pcs_fence_kdump <image timestamp>
+# return 0 if fence_kdump is configured in Pacemaker cluster and kdump initrd needs to be rebuilt
+function check_pcs_fence_kdump()
 {
 	local image_time=$1
 	local cib_time
@@ -239,8 +239,8 @@ function check_rebuild()
 		image_time=0
 	fi
 
-	#also rebuild when cluster conf is changed and fence kdump is enabled.
-	check_fence_kdump $image_time && modified_files="cluster-cib"
+	#also rebuild when Pacemaker cluster conf is changed and fence kdump is enabled.
+	check_pcs_fence_kdump $image_time && modified_files="cluster-cib"
 
 	EXTRA_BINS=`grep ^kdump_post $KDUMP_CONFIG_FILE | cut -d\  -f2`
 	CHECK_FILES=`grep ^kdump_pre $KDUMP_CONFIG_FILE | cut -d\  -f2`
-- 
1.8.3.1

